### PR TITLE
community: Fix planner_prompt so api_controller defines "action"

### PR DESCRIPTION
**Pull request by julesterrien**
**Description:** 
This PR proposes the same fix initially created by @eunhye1kim [here](https://github.com/eunhye1kim/langchain/commit/f7a5ecc077e15e6a33824439f2d925dcb0fcc242) so it can be merged in the main lib

**Issue:** 
https://github.com/langchain-ai/langchain/issues/24382
https://github.com/langchain-ai/langchain/issues/22545

**Dependencies:** 

**Twitter handle:** 
@julesterrien

### CODE REVIEW

**Comment by julesterrien** [comment]
@eunhye1kim i didn't see if you posted a PR with your proposed fix into the langchain repo so went ahead and opened this one so the fix can be merged. If you have an open PR let me know

**Comment by eunhye1kim** [comment]
> @eunhye1kim i didn't see if you posted a PR with your proposed fix into the langchain repo so went ahead and opened this one so the fix can be merged. If you have an open PR let me know

The PR(https://github.com/langchain-ai/langchain/pull/24384) has already been merged after receiving a fix from the maintainer. Therefore, if additional changes are required regarding the PR, they must be posted separately, including in this PR.

Additionally, I received opposition to modifying the prompt in comments from administrators. Although, I solved the problem by modifying the prompt in the library code in my virtualenv environment due to scheduling issues. But I think adding a custom prompt option might solve the problem.

**Comment by julesterrien** [comment]
@eunhye1kim 👍 yep updating the prompt to say "must" instead of "should" as you suggested also solved it for me 

maintainers: what alternative fix do you suggest if not this one?

File: libs/community/langchain_community/agent_toolkits/openapi/planner_prompt.py [hunk 1]
```diff
@@ -84,7 +84,7 @@
 
 Plan: the plan of API calls to execute
 Thought: you should always think about what to do
-Action: the action to take, should be one of the tools [{tool_names}]
+Action: the action to take, 'must be' one of the tools [{tool_names}] without additional words
```
Comment (Undertone0809) [inline libs/community/langchain_community/agent_toolkits/openapi/planner_prompt.py:87]: I wonder why 'must be' add quotation marks.

**Comment by julesterrien** [comment]
@baskaryan @efriis @eyurtsev @ccurme @vbarda @hwchase17

File: libs/community/langchain_community/agent_toolkits/openapi/planner_prompt.py (hunk 1 above)
Comment (julesterrien) [reply libs/community/langchain_community/agent_toolkits/openapi/planner_prompt.py:87]: Tbh i'm not sure and copied the initial fix [here](https://github.com/eunhye1kim/langchain/commit/f7a5ecc077e15e6a33824439f2d925dcb0fcc242) 

I assume this just adds extra emphasis for the LLM to respect this condition -- somewhat similar to saying: "Important: ..."

**Pull request closed without merging**